packages/jest-runner/src/index.ts

```diff
@@ -26,16 +26,15 @@ interface WorkerInterface extends Worker {
   worker: typeof worker;
 }
 
+export {CallbackTestRunner, EmittingTestRunner} from './types';
```

Comment:
Not `type`?

Reply:
`type` only allows `implements`. Which means that constructor is lost. I think not is nice to inherit it, but that is possible only with `extends`. If class is exported (like I did here), both `extends` and `implements` are allowed.

I did not knew that. Found it out while writing type tests ;D

Reply:
👍👍